packages/-ember-data/tests/unit/debug-test.js

```diff
@@ -1,108 +1,106 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
 import { computed } from '@ember/object';
-import { createStore } from 'dummy/tests/helpers/store';
+import { setupTest } from 'ember-qunit';
 
 import { module, test } from 'qunit';
 
-import DS from 'ember-data';
-
-const TestAdapter = DS.Adapter.extend();
-
-module('Debug');
-
-test('_debugInfo groups the attributes and relationships correctly', function(assert) {
-  const MaritalStatus = DS.Model.extend({
-    name: DS.attr('string'),
-  });
-
-  const Post = DS.Model.extend({
-    title: DS.attr('string'),
-  });
-
-  const User = DS.Model.extend({
-    name: DS.attr('string'),
-    isDrugAddict: DS.attr('boolean'),
-    maritalStatus: DS.belongsTo('marital-status', { async: false }),
-    posts: DS.hasMany('post', { async: false }),
-  });
-
-  let store = createStore({
-    adapter: TestAdapter.extend(),
-    maritalStatus: MaritalStatus,
-    post: Post,
-    user: User,
-  });
-
-  let record = store.createRecord('user');
-
-  let propertyInfo = record._debugInfo().propertyInfo;
-
-  assert.equal(propertyInfo.groups.length, 4);
-  assert.equal(propertyInfo.groups[0].name, 'Attributes');
-  assert.deepEqual(propertyInfo.groups[0].properties, ['id', 'name', 'isDrugAddict']);
-  assert.equal(propertyInfo.groups[1].name, 'belongsTo');
-  assert.deepEqual(propertyInfo.groups[1].properties, ['maritalStatus']);
-  assert.equal(propertyInfo.groups[2].name, 'hasMany');
-  assert.deepEqual(propertyInfo.groups[2].properties, ['posts']);
-});
-
-test('_debugInfo supports arbitray relationship types', function(assert) {
-  const MaritalStatus = DS.Model.extend({
-    name: DS.attr('string'),
-  });
-
-  const Post = DS.Model.extend({
-    title: DS.attr('string'),
-  });
-
-  const User = DS.Model.extend({
-    name: DS.attr('string'),
-    isDrugAddict: DS.attr('boolean'),
-    maritalStatus: DS.belongsTo('marital-status', { async: false }),
-    posts: computed(() => [1, 2, 3])
-      .readOnly()
-      .meta({
-        options: { inverse: null },
-        isRelationship: true,
-        kind: 'customRelationship',
-        name: 'posts',
-        type: 'post',
-      }),
-  });
-
-  let store = createStore({
-    adapter: TestAdapter.extend(),
-    maritalStatus: MaritalStatus,
-    post: Post,
-    user: User,
+module('Debug', function(hooks) {
+  setupTest(hooks);
+
+  test('_debugInfo groups the attributes and relationships correctly', function(assert) {
+    const MaritalStatus = Model.extend({
+      name: attr('string'),
+    });
+
+    const Post = Model.extend({
+      title: attr('string'),
+    });
+
+    const User = Model.extend({
+      name: attr('string'),
+      isDrugAddict: attr('boolean'),
+      maritalStatus: belongsTo('marital-status', { async: false }),
+      posts: computed(() => [1, 2, 3])
```

Comment:
this got copied from the test below but should still be a normal hasMany relationship